### ISSUE TITLE
Teleport Access Slackbot 0.1.0-alpha.1:teleport-jira-v0.1.0-alpha.1-0-gebfacdf 

### ISSUE THREAD

**Issue by benarent** [gravitational/teleport-plugins#91]
The `teleport-slack`  has jira version in INFO when running latest binary. 

```
teleport-421-auth ~/teleport-access-slack: teleport-slack start
INFO   Starting Teleport Access Slackbot 0.1.0-alpha.1:teleport-jira-v0.1.0-alpha.1-0-gebfacdf slack/main.go:145
```

**Comment by webvictim** [comment]
This is also the case for `mattermost` and `pagerduty`. I fixed `mattermost`, but `pagerduty` and `slack` don't work yet.

This is a byproduct of the fact that we used to only cut one release from a given branch. Now we're cutting more than one, our previous strategy for getting version information (`git describe --dirty --long --tags`) won't work for multiple tags.

Needs some thought.

**Comment by webvictim** [comment]
Fixed in `v0.1.0-alpha.2`
